api/handler/delete.go

```diff
@@ -77,6 +87,41 @@ func (h *handler) DeleteObjectHandler(w http.ResponseWriter, r *http.Request) {
 			zap.Error(err))
 	}
 
+	m := &layer.SendNotificationsParams{
+		Event: notifications.EventObjectRemovedDeleteMarkerCreated,
```

Comment:
We shouldn't produce such notification type on every object deletion (see comment in `layer.deleteObject` function).